Services/UI/OJS.Services.Ui.Business/Implementations/SubmissionsBusinessService.cs

```diff
@@ -357,6 +366,15 @@ public async Task Submit(SubmitSubmissionServiceModel model)
                 currentUser.Id!,
                 model.Official);
 
+        if (participant is not null &&
+            participant.Contest.CantSubmitConcurrently &&
+            participant.Submissions.Any(s => s.Processed != true))
+        {
+            throw new BusinessServiceException(
```

Comment:
Do not throw Exceptions like that. Better use one of the validator's services in UI/OJS.Service.UI.Business/Validators/Implementations/Submissions or simply create a new one for your logic.

Reply:
Fixed